### PR TITLE
Add Azure Servicebus Topic Source e2e test

### LINKED ISSUE #132: e2e: Tests for Azure Service Bus Topic e2e

### PR DESCRIPTION
**Pull request by FranBarrera**
Closes #132

### CODE REVIEW

**Review by cab105 — COMMENTED** [review]
Realized #136 brings in the changes from this PR, and my comments have been left there. Will approve both once the other is ready to go.

File: test/e2e/framework/azure/servicebus.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/servicebus/mgmt/servicebus"
+	sv "github.com/Azure/azure-service-bus-go"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/triggermesh/test-infra/test/e2e/framework"
+)
+
+// CreateNamespaceClient will create the servicebus client
+func CreateNamespaceClient(ctx context.Context, subscriptionID, region string) *servicebus.NamespacesClient {
```
Comment (odacremolbap) [inline test/e2e/framework/azure/servicebus.go:30]: Can this 3 functions
- CreateNamespaceClient
- CreateNamespace
- CreateNsService

be used at `CreateEventHubComponents` replacing the Namespace management [in there](https://github.com/triggermesh/test-infra/blob/bb40088e1bd52233e56988133b2cb3d9c06596e7/test/e2e/framework/azure/eventhub.go#L30)?

I think that should be a drop-in

**Review by odacremolbap — APPROVED** [review]
+1 with a couple comments

File: test/e2e/framework/azure/servicebus.go (hunk 1 above)
Comment (FranBarrera) [reply test/e2e/framework/azure/servicebus.go:30]: It sounds good, I'll try to improve it after the "first release" of the azure tests.

**Pull request merged**